pipelines/jobs/configurations/jdk11u.groovy

```diff
@@ -1,15 +1,15 @@
 targetConfigurations = [
-        'x64Mac'        : [    'temurin',    'openj9'                    ],
```

Comment:
Did you mean to delete all the other variants from the standard nightly?

Reply:
up in https://github.com/adoptium/ci-jenkins-pipelines/issues/543  for detail discussion